pyproject.toml

```diff
@@ -63,18 +63,6 @@ requires = ["pdm-backend"]
 build-backend = "pdm.backend"
 
 [tool.pdm.dev-dependencies]
-dev = [
```

Comment:
`dev` group was present 2 times. It was removed automatically when running `pdm add --dev ipython`

Reply:
Must have been messed up when pdm moved it to use the new "dependency-groups" section. Looks like the `standalone` group has been duplicated too.